src/zjs_modules.c

```diff
@@ -123,8 +123,40 @@ static jerry_value_t native_require_handler(const jerry_value_t function_obj,
             return mod->instance;
         }
     }
+    ZJS_PRINT("Native module not found, searching for JavaScript module %s\n", module);
```

Comment:
I generally agree with that, but this only shows up at build time - I think it should print no matter what so people can look at the build log to see what got included.  Thoughts?